packages/manager/modules/ovh-reket/src/index.js

```diff
@@ -0,0 +1,74 @@
+import { Reket } from '@ovhcloud/reket-core';
+import { AxiosReketClient } from '@ovhcloud/reket-axios-client';
+
+import { DEFAULT_REQUEST_TYPES } from './constants';
+import { redirectToLoginPage, redirectToLogoutPage } from './redirections';
+
+export const ssoAuthHookFn = (error) => {
+  const rejectedError = error;
+
+  if (
+    rejectedError.status === 403 &&
+    (rejectedError.data.message === 'This session is forbidden' ||
+      rejectedError.data.message === 'This session is invalid')
+  ) {
+    // consider a 401 if previous case is matched
+    rejectedError.status = 401;
+  }
+
+  // Redirect on 401
+  if (rejectedError.status === 401 && !rejectedError.config.preventLogout) {
+    redirectToLogoutPage();
+    return Promise.reject(rejectedError);
+  }
+
+  // If CODE 471 AKA Low-order session
+  if (rejectedError.status === 471) {
+    redirectToLoginPage();
+    return Promise.reject(rejectedError);
+  }
+
```

Comment:
Don't know if this is useful to keep or not:
https://github.com/ovh/manager/blob/8d658acff9688f923fe617788bb50c104273c6b2/packages/components/ng-ovh-sso-auth/src/factory.js#L106-L110